code/controllers/subsystem/nightshift.dm

```diff
@@ -0,0 +1,59 @@
+SUBSYSTEM_DEF(nightshift)
+	name = "Night Shift"
+	init_order = INIT_ORDER_NIGHTSHIFT
+	priority = FIRE_PRIORITY_NIGHTSHIFT
```

Comment:
Sure these are necessary? TG just let these be the default values.

Reply:
They aren't strictly necessary, but I think for the sake of consistency and look-good, we should make every subsystem it's own init order and priority define - even if they're the same value as the default.

Reply:
In that case, I believe the number values are not the same as default.

Reply:
So - the priority, I can change, I just figured that it'd be better to assign less of a tick to a controller that only fires once every 600 ticks anyways. However, the init order was trying to do some futureproofing - when we convert the lighting controller over, this should initialize after lighting and power, to avoid any shenanigans with lights being changed before they're initialized.